### PR TITLE
Chapter 7 Sign Up

### PR DESCRIPTION
**Pull request by Iskandar19**

### CODE REVIEW

**Comment by Iskandar19** [comment]
ready

File: app/assets/stylesheets/users.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the Users controller here.
+// They will automatically be included in application.css.
```
Comment (ngocvt-0484) [inline app/assets/stylesheets/users.scss]: delete this file because not use

File: config/routes.rb [hunk 2]
```diff
@@ -1,4 +1,10 @@
 Rails.application.routes.draw do
+  get 'users/show'
```
Comment (ngocvt-0484) [inline config/routes.rb]: remove it

File: test/controllers/users_controller_test.rb [hunk 3]
```diff
@@ -0,0 +1,13 @@
+require "test_helper"
+
+class UsersControllerTest < ActionDispatch::IntegrationTest
+  test "should get show" do
+    get users_show_url
```
Comment (ngocvt-0484) [inline test/controllers/users_controller_test.rb]: delete this file
you dont need do test

**Comment by Iskandar19** [comment]
ready

**Comment by ngocvt-0484** [comment]
merged

**Pull request merged**